src/utils/process.js

```diff
@@ -65,30 +65,49 @@ async function findProcessUnix (browserId) {
     return findProcessIdUnix(browserId, output);
 }
 
-async function checkUnixProcessIsKilled (processId) {
+async function isUnixProcessKilled (processId) {
     const output = await getProcessOutputUnix();
 
-    if (isProcessExistUnix(processId, output)) {
-        await delay(CHECK_KILLED_DELAY);
+    return !isUnixProcessExist(processId, output);
+}
 
-        await checkUnixProcessIsKilled();
-    }
+async function killUnixProcessSoft (processId) {
+    process.kill(processId);
+}
+
+async function killUnixProcessHard (processId) {
+    process.kill(processId, HARD_KILL_FLAG);
 }
 
 async function killProcessUnix (processId) {
-    let timeoutError = false;
+    const maxSoftTries = 2;
+    let softTries = 0;
+    let unixProcessKilled = false;
 
-    process.kill(processId);
+    do {
+        await killUnixProcessSoft(processId);
 
-    const killTimeoutTimer = delay(CHECK_PROCESS_IS_KILLED_TIMEOUT)
-        .then(() => {
-            timeoutError = true;
-        });
+        softTries++;
 
-    return Promise.race([killTimeoutTimer, checkUnixProcessIsKilled(processId)]).then(() => {
-        if (timeoutError)
-            throw new Error(killProcessTimeoutError);
-    });
+        await delay(CHECK_KILLED_DELAY);
+
+        unixProcessKilled = await isUnixProcessKilled(processId);
+    } while (!unixProcessKilled && softTries < maxSoftTries);
```

Comment:
We use the `stroustoup` style for code blocks: https://eslint.org/docs/rules/brace-style#stroustrup

```suggestion
    do {
        await killUnixProcessSoft(processId);

        softTries++;

        await delay(CHECK_KILLED_DELAY);

        unixProcessKilled = await isUnixProcessKilled(processId);
    } 
    while (!unixProcessKilled && softTries < maxSoftTries)
```